db-engine/pom.xml

```diff
@@ -237,6 +237,19 @@
                     </execution>
                 </executions>
             </plugin>
+             <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>2.19.1</version>
+                        <configuration>
+                            <additionalClasspathElements>
+                                <additionalClasspathElement>path/to/additional/resources</additionalClasspathElement>
```

Comment:
What I'm saying is additionalClasspathElement should receive an path, and "path/to/additional/resources" or "path/to/additional/jar" don't look like real paths.